src/notification/notification.service.ts

```diff
@@ -364,4 +365,39 @@ export class NotificationService {
             ])
             .delete();
     }
+
+    async createMentionNotification(mention: Mention) {
```

Comment:
Product Question: Should we create a notification when someone mentions their own account in a post?

If not, we should add a check for that. 

```
if (post.author.id === accountId) {
            // Do not create a notification if author mentions their own account
            return;
        }
```

Reply:
Yeah good one! added in 731d4d4d24dd984e00f9b9bead0a6c1ac8b463d8